src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/helpers/TestAssessmentRepository.kt

```diff
@@ -8,5 +8,5 @@ import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.entity.Offende
 @Repository
 interface TestAssessmentRepository : JpaRepository<Assessment, Long> {
   fun findByOffender(offender: Offender): List<Assessment>
-  fun findByOffenderPrisonNumber(prisonNumber: String): List<Assessment>
+  fun findByOffenderPrisonNumberOrderById(prisonNumber: String): List<Assessment>
```

Comment:
didn't know this existed 👍

Reply:
Yes I added it, under the logic of if the repo code is for testing only should not be in production code. It's a nice way of separating test from prod code.. otherwise we end up with a lot of methods but only some used in prod code.